python/rmm/docs/guide.md

```diff
@@ -187,3 +187,108 @@ allocator.
 
 >>> torch.cuda.memory.change_current_allocator(rmm_torch_allocator)
 ```
+
+
+
+## Memory statistics and profiling
+
+RMM has a tool for tracking memory statistics and memory profiling. It can be enabled in two ways:
+  - Use the context manager `rmm.statistics.statistics()` to enable statistics tracking for a specific code block.
+  - Call `rmm.statistics.enable_statistics()` to enable statistics tracking globally.
+
+Common to both usages is that they modify the currently active RMM memory resource. A `StatisticsResourceAdaptor` is pushed onto the current RMM memory resource stack and must remain the topmost resource throughout the statistics tracking:
```

Comment:
Is it true that RMM Python has a stack of memory resources? I looked and I can't find this stack in the source, so I'm not sure this is correct.

Reply:
Not really, except in the sense that adapter resources contain their parent, which you might think of a bit like a stack.